Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py

```diff
@@ -3057,6 +3063,28 @@ def condition(self, fragment, stepList, key, hasHarvest):
         'Era' : 'Phase2C17I13M9',
         'ScenToRun' : ['GenSimHLBeamSpot','DigiTrigger','RecoGlobal', 'HARVESTGlobal', 'ALCAPhase2'],
     },
+   '2026D111' : {
+        'Geom' : 'Extended2026D111',
+        'HLTmenu': '@relval2026',
+        'GT' : 'auto:phase2_realistic_T36',
+        'Era' : 'Phase2C22I13M9',
+        'ScenToRun' : ['GenSimHLBeamSpot','DigiTrigger','RecoGlobal', 'HARVESTGlobal', 'ALCAPhase2'],
+    },
+    '2026D112' : {
+        'Geom' : 'Extended2026D112',
+        'HLTmenu': '@relval2026',
+        'GT' : 'auto:phase2_realistic_T37',
+        'Era' : 'Phase2C22I13M9',
+        'ScenToRun' : ['GenSimHLBeamSpot','DigiTrigger','RecoGlobal', 'HARVESTGlobal', 'ALCAPhase2'],
+    },
+    '2026D113' : {
+        'Geom' : 'Extended2026D113',
+        'HLTmenu': '@relval2026',
+        'GT' : 'auto:phase2_realistic_T38',
+        'Era' : 'Phase2C22I13M9',
+        'ScenToRun' : ['GenSimHLBeamSpot','DigiTrigger','RecoGlobal', 'HARVESTGlobal', 'ALCAPhase2'],
+    },
+}
```

Comment:
Extra `}` ?

Reply:
Removed } on line 3088, was mistakenly inserted in an incorrect squash attempt, thanks for the flash review!